docs/ui.md

```diff
@@ -14,7 +14,7 @@ The Connection Dialog is mainly defined in the Tableau Custom Dialog (.tcd) file
 
 The connector is displayed as "[Display Name] by [Company Name]" in the connection dialog and connection list.
 
-"For support, contact [Company Name]" is displayed at the bottom left of the connector. Clicking on this link will send the user to the support link defined in the manifest. This link also displays in error messages.
+"For support, contact [Company Name]" is displayed at the bottom left of the connector. Clicking on this link will send the user to the support link defined in the manifest. This link also displays in error messages. The support link must be https to be packaged into a taco file.
```

Comment:
for the second sentence, remove "on" and "will". "Clicking this link sends the user to the support link..."
for the fourth sentence, cap https and format taco: "The support link must use HTTPS to be packaged into a `.taco` file.